mongodbatlas/data_source_mongodbatlas_cloud_provider_snapshot_restore_job.go

```diff
@@ -76,7 +76,7 @@ func dataSourceMongoDBAtlasCloudProviderSnapshotRestoreJob() *schema.Resource {
 				Computed: true,
 			},
 			"oplog_ts": {
-				Type:     schema.TypeString,
+				Type:     schema.TypeInt,
```

Comment:
Was this a bug found?

Reply:
It was a typo it seems, it should have been Int since it gave me an error of type so I changed it

Reply:
In the underlying docs it's a string https://docs.atlas.mongodb.com/reference/api/cloud-backup/restore/create-one-restore-job/ so we may want to ensure it's not a bigger bug or a miss of something else.

Reply:
mm weird, to me appears is a integer for `oplogTs`

Reply:
Ah, I think the docs are wrong.  In the POST it has it as a string but in the GET it shows an integer, which makes more sense.  I think this fix is right and docs wrong (which test caught!). Let's leave it in but document in the PR description (not the title, just the description is fine)

Reply:
Ignore previous comment - string in docs in both places.  Could Terraform be storing the string as an int?  Seems weird.